src/components/tl/mlx5/mcast/tl_mlx5_mcast_coll.c

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
```

Comment:
@MamziB Since you are porting some code, do you reflect that on the copyright as well?

Reply:
If the ported code is proprietary Nvidia, ex. Mellanox code, does it need to be reflected?